src/calliope/math/base.yaml

```diff
@@ -874,27 +873,33 @@ global_expressions:
   cost_investment:
     title: Total investment costs
     description: >-
-      The installation costs of a technology, including annualised investment costs and annual maintenance costs.
+      The installation costs of a technology, including those linked to the nameplate capacity, land use, storage size, and binary/integer unit purchase.
     default: 0
     unit: cost
     foreach: [nodes, techs, costs]
     where: >-
       cost_investment_flow_cap OR cost_investment_storage_cap OR cost_investment_source_cap OR
       cost_investment_area_use OR cost_investment_purchase
     equations:
-      - expression: >
-          $annualisation_weight * (
-            ($depreciation_rate + cost_om_annual_investment_fraction) * (
-              sum(default_if_empty(cost_investment_flow_cap, 0), over=carriers) +
-              default_if_empty(cost_investment_storage_cap, 0) +
-              default_if_empty(cost_investment_source_cap, 0) +
-              default_if_empty(cost_investment_area_use, 0) +
-              default_if_empty(cost_investment_purchase, 0)
-            )
-            + sum(cost_om_annual * flow_cap, over=carriers)
-          )
+      - expression: >-
+          sum(default_if_empty(cost_investment_flow_cap, 0), over=carriers) +
+          default_if_empty(cost_investment_storage_cap, 0) +
+          default_if_empty(cost_investment_source_cap, 0) +
+          default_if_empty(cost_investment_area_use, 0) +
+          default_if_empty(cost_investment_purchase, 0)
+
+  cost_investment_annualised:
+    title: Equivalent annual investment costs
+    description: >-
+      An annuity factor has been applied to scale lifetime investment costs to annual values that can be directly compared to operation costs.
```

Comment:
Maybe the use of `annualisation_weight` could be included here, to prevent someone taking the _[...] to annual values [...]_ too literally?

Example:

> An annuity factor has been applied to scale lifetime investment costs to annual values that can be directly compared to operation costs. If the modeling period is not equal to one full year, this will be scaled accordingly.